script.py

```diff
@@ -1,73 +1,67 @@
-import time 
-from selenium import webdriver 
+import time
+from selenium import webdriver
 from selenium.webdriver.support.select import Select
 
-driver = webdriver.Chrome()
+driver = webdriver.Chrome('D:/webdriver/chromedriver.exe')
```

Comment:
This will not work in other Os. You can remove this path

Reply:
ok